lib/carin_for_blue_button_test_kit/custom_groups/visual_inspection_and_attestation/v200_client/authentication.rb

```diff
@@ -11,20 +11,43 @@ class CARIN4BBAuthTest < Inferno::Test
     verifies_requirements 'hl7.fhir.us.carin-bb_2.0.0@1',
                           'hl7.fhir.us.carin-bb_2.0.0@48'
 
+    input :c4bb_authentication_options,
+          title: 'Uses appropriate authentication',
+          description: %(
+            I attest that the Health IT Module
```

Comment:
Change "I attest" to "The developer of ... attests that the system" here and across the repo